guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
```

Comment:
<ol><li>To state it succinctly, PointCNN differs from conventional <s>grid based</s> <strong>grid-based</strong> CNNs primarily due to the application of <s>X-Conv layers;</s> <strong>X-Conv layers.</strong> <strong>E</strong>ven then, the general process is</li><li><strong>T</strong>he way the local regions are extracted</li><li><strong>T</strong>he way the information from local regions is learned</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
''corrected accordingly'', Thanks.

Reply:
<p>You can consider including:</p><p>1) <s>grid -based</s>  <strong>grid-based</strong> (extra space in between)</p>

Reply:
no extra space, were found, in HTML or notebook, might be due to ReviewNB.